operator/docker/base/Dockerfile

```diff
@@ -67,53 +58,3 @@ ARG TARGETARCH
 COPY --from=builder /cass-operator/build/bin/cass-operator-linux-${TARGETARCH} bin/operator
 
 CMD [ "/go/bin/operator" ]
-
-# UBI Image
-
-
-
-#############################################################
-
-FROM ${BASE_OS} AS builder-ubi
```

Comment:
Are we building the UBI images elsewhere? If not then I am pretty sure we still need this. It is needed for OpenShift integration.

Reply:
What is our Openshift strategy? As far as I read from the cass-operator tickets, it hasn't been working for a while:

https://github.com/datastax/cass-operator/issues/266

So not in 4.5, not in 4.6, probably not in 4.7 .. and in k8ssandra we haven't tested anything against Openshift. Also, this k8ssandra/cass-operator path is not supported in the container catalog (only datastax/cass-operator is).

Reply:
@bradfordcp can you chime in here?

Reply:
We can add back or update needed functionality in follow up PRs. I don't want to hold up merging on this as it currently does not impact k8ssandra.